.env.production

```
@@ -4,6 +4,7 @@ ENVIRONMENT=PROD
 API_URL_PREFIX='https://app-backend.k8s.test.cd.teamdigitale.it'
 # PagoPA RESTful API
 PAGOPA_API_URL_PREFIX='https://wisp2.pagopa.gov.it/pp-restapi-CD'
+PAGOPA_API_URL_PREFIX_QA='https://wisp2.pagopa.gov.it/pp-restapi-CD'
```

I'd call it _TEST for consistency with the param we pass to the backend

```suggestion
PAGOPA_API_URL_PREFIX_TEST='https://wisp2.pagopa.gov.it/pp-restapi-CD'
```

---

you will also find the URL of the test PM from this file's revision a couple months ago (before we switched to prod)